### ISSUE TITLE
PDFUA-178: Figure with no Alt text at all | FAIL

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/PDFA-TEST#175]
Jira issue originally created by user @:

This example demonstrates the failure condition that applies for an image that conveys information, but [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) is missing.

**Comment by zakkinsey** [comment]
Comment created by @BPeboeck:

I added Matterhorn Protocol 13-005 concerning procedure step 4.
